singularity-opac/photons/mean_photon_variant.hpp

```diff
@@ -92,6 +92,27 @@ class MeanVariant {
         opac_);
   }
 
+  PORTABLE_INLINE_FUNCTION
+  Real AbsorptionCoefficient(const Real rho, const Real temp,
+                            const int gmode = 0) const {
```

Comment:
```suggestion
                            const int gmode = Rosseland) const {
```
Is this change OK to make? There are a few other places with integers as default args to these functions here and below.

Reply:
For some reason my comment got duplicated, I deleted the duplication, but that deleted it here - I made the change, but asked about `non_cgs_photons.hpp`, which doesn't include this as a header, but has these `gmode` values also defaulted to integers. Should the enum be moved to a separate header to be included here and in `non_cgs_photons.hpp`?

Reply:
@brryan - any thoughts on this header question? Thanks!

Reply:
@RyanWollaeger Sorry I'm not sure how I missed this question. Yeah I think you're right, we need a new header file, `photon_types.hpp` or something (you may have a better name in mind) to be included in both these existing files.

Reply:
Should this be mimicked in neutrinos too? It might be useful to have rosseland vs planck mean in neutrinos too

Reply:
Just made this update for photons - will map this over to neutrinos shortly, along with the other AbsorptionCoefficient/Emissivity changes.

Reply:
We can try out just the photons for now and see if anything goes wrong bringing it into artemis before typing out the neutrinos